application/cms/models.py

```diff
@@ -937,4 +939,71 @@ class LowestLevelOfGeography(db.Model):
     description = db.Column(db.String(255), nullable=True)
     position = db.Column(db.Integer, nullable=False)
 
-    pages = relationship("Page", back_populates="lowest_level_of_geography")
+    pages = db.relationship("Page", back_populates="lowest_level_of_geography")
+
+
+class Topic(db.Model):
+    __tablename__ = "topic"
+
+    id = db.Column(db.Integer, primary_key=True)
+    uri = db.Column(db.String(64), nullable=False)
+    title = db.Column(db.String(255), nullable=False)
+    description = db.Column(db.Text, nullable=True)  # a sentence below topic heading on homepage
+    additional_description = db.Column(db.TEXT, nullable=True)  # short paragraph displayed on topic page
+
+    # Find a list of subtopics that belong to this topic using this relationship
+    subtopics = db.relationship("Subtopic", back_populates="topic")
+
+
+subtopic_measure = db.Table(
+    "subtopic_measure",
+    db.Column("subtopic_id", db.Integer, db.ForeignKey("subtopic.id"), primary_key=True),
+    db.Column("measure_id", db.Integer, db.ForeignKey("measure.id"), primary_key=True),
+)
+
+
+class Subtopic(db.Model):
+    __tablename__ = "subtopic"
+
+    id = db.Column(db.Integer, primary_key=True)
+    uri = db.Column(db.String(64), nullable=False)
+    title = db.Column(db.String(255), nullable=False)
+    position = db.Column(db.Integer, default=0)  # for ordering on the page
+    topic_id = db.Column(db.Integer, ForeignKey("topic.id"), nullable=True)
+
+    topic = db.relationship("Topic")
```

Comment:
I _think_ you need `back_populates` on both sides of the relationship for it to be bidirectionally useable in Python.